api/lagoon/client/_lgraphql/projects/projectByNameExtended.graphql

```diff
@@ -0,0 +1,36 @@
+query (
+  $name: String!) {
+  projectByName(
+    name: $name) {
+    id
+    name
+    gitUrl
+    subfolder
+    routerPattern
+    branches
+    pullrequests
+    problemsUi
+    factsUi
+    productionEnvironment
+    deploymentsDisabled
+    deployTargetConfigs{
+      id
+      deployTarget{
+        id
+        name
+        token
+      }
+    }
```

Comment:
Is this needed at all here?

Also retrieving the token will fail for all users except `platform-owner`

Reply:
Ah forgot to clean this up, just grabbed it from the existing cmd fragment.